scripts/i18n-extract-keys.ts

```
@@ -0,0 +1,152 @@
+  const oldFilePath = filePath.replace('.json', '_old.json');
+  const fileToRead = fs.existsSync(oldFilePath) ? oldFilePath : filePath;
+
+  const currentTranslations = fs.existsSync(fileToRead)
+    ? cleanDeep(JSON.parse(fs.readFileSync(fileToRead, 'utf-8')))
+    : {};
+
+  const fileKeys = allKeys[path.basename(filePath, '.json')] ?? {};
+
+  const newTranslations = JSON.parse(JSON.stringify(fileKeys)); // deep clone
+
+  const merge = (target: any, source: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)